deepinv/optim/data_fidelity.py

```diff
@@ -578,6 +578,33 @@ def prox(
         return t
 
 
+class IntensityLoss(DataFidelity):
+    r"""
+    Intensity loss as the data fidelity term for phase retrieval reconstrunction.
+
+    In this case, the data fidelity term is defined as
+
+    .. math::
+
+        f(x) = \sum_{i=1}^{m}{||a_i x|^2-y_i|^2},
+
+    where :math:`a_i` is the i-th row of the measurement matrix :math:`A` and :math:`y_i` is the i-th entry of the measurements, and :math:`m` is the number of measurements.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, y, physics, *args, **kwargs):
+        return torch.sum(torch.pow(physics(x) - y, 2))
+
+    def grad(self, x, y, physics, *args, **kwargs):
+        y_est = physics(x)
+        y_est_bis = physics.A(x)
+        diff = y - y_est
+        return -2 * physics.A_adjoint(y_est_bis * diff)
```

Comment:
```suggestion
        return physics.A_adjoint(y_est_bis * diff)
```

Reply:
The negative sign is forgotten here, I have changed the computation a bit so the final formula doesn't need the negative sign.